specs/sequencer-relayer.md

```diff
@@ -0,0 +1,152 @@
+# Sequencer-Relayer
+
+## Overview
+
+The sequencer-relayer, a.k.a relayer, is an application whose primary
+responsibility is to store sequenced rollup data on a remote data availability
```

Comment:
```suggestion
responsibility is to publish sequenced rollup data on a remote data availability
```

Reply:
Done in [045e82c](https://github.com/astriaorg/astria/pull/1126/commits/045e82c6be5afcd9e2fd53055e3d977e4037ecbf).